scipy/sparse/linalg/eigen/lobpcg/tests/test_lobpcg.py

```diff
@@ -216,14 +217,54 @@ def test_hermitian():
         else:
             B = np.random.rand(size, size) + 1.j * np.random.rand(size, size)
             B = 10 * np.eye(size) + B.dot(B.T.conj())
-            w, v = lobpcg(H, X, B, maxiter=5000)
+            w, v = lobpcg(H, X, B, maxiter=5000, largest=False)
             w0, v0 = eigh(H, B)
 
         for wx, vx in zip(w, v.T):
             # Check eigenvector
-            assert_allclose(np.linalg.norm(H.dot(vx) - B.dot(vx) * wx) / np.linalg.norm(H.dot(vx)),
+            assert_allclose(np.linalg.norm(H.dot(vx) - B.dot(vx) * wx)
+                            / np.linalg.norm(H.dot(vx)),
                             0, atol=5e-4, rtol=0)
 
             # Compare eigenvalues
             j = np.argmin(abs(w0 - wx))
             assert_allclose(wx, w0[j], rtol=1e-4)
+
+def test_eigs_consistency():
+    n = 20
+    vals = [np.arange(n, dtype=np.float64) + 1]
```

Comment:
This can be done via 
```python
vals = np.arange(1, n+1, dtype=float)
```
Similarly for n=5 below

Reply:
Thanks, that is what careless copy-pasting without reading does :)